### PR TITLE
Add find command

### PR DESCRIPTION
**Pull request by bynect**
For now it can only find the first matching substring in the file, and it is fairly slow in big files, but it is doing well its job for what I have tried right now

### CODE REVIEW

**Comment by bynect** [comment]
A possible improvement would be to a state in the find function and when no input was provided continue searching the old string from the saved position

File: src/config_dialog.c [hunk 1]
```diff
@@ -113,6 +113,24 @@ static void read_only_cmd(char **words, unsigned int words_len) {
...
+static void find(char **words, unsigned int words_len) {
+    if (words_len == 1) {
+        unsigned int len = strlen(words[0]);
+        int index;
+        for (unsigned int at = 0; at < num_lines && (at != num_lines); ++at) {
+            if (lines[at].length >= len && (index = uchar32_sub(lines[at].data, words[0], lines[at].length, len)) >= 0) {
+                cursor.y = at;
+                cursor.x = index;
+                cursor_in_valid_position();
+                syntaxHighlight();
```
Comment (arthurbacci) [inline src/config_dialog.c:130]: Why this call to `syntaxHighlight()`?

Comment (bynect) [reply src/config_dialog.c:130]: I added it for error, it is not actually needed

Comment (bynect) [reply src/config_dialog.c:130]: Its needed because syntaxHighlight has to be updated every time the lines shown on screen change

**Pull request merged**